src/zenml/integrations/tekton/orchestrators/tekton_orchestrator.py

```diff
@@ -282,65 +289,100 @@ def _configure_container_op(
                 host_path=host_path,
             )
             logger.debug(
-                "Adding host path volume for the local ZenML stores (path: %s) "
-                "in Tekton pipelines container.",
+                "Adding host path volume for the local ZenML stores "
+                "(path: %s) in Tekton pipelines container. ",
                 local_stores_path,
             )
 
-            if sys.platform == "win32":
-                # File permissions are not checked on Windows. This if clause
-                # prevents mypy from complaining about unused 'type: ignore'
-                # statements
-                pass
-            else:
-                # Run KFP containers in the context of the local UID/GID
-                # to ensure that the local stores can be shared
-                # with the local pipeline runs.
-                container_op.container.security_context = (
-                    k8s_client.V1SecurityContext(
-                        run_as_user=os.getuid(),
-                        run_as_group=os.getgid(),
-                    )
-                )
-                logger.debug(
-                    "Setting security context UID and GID to local user/group "
-                    "in Tekton pipelines container."
-                )
-
-            container_op.container.add_env_variable(
-                k8s_client.V1EnvVar(
-                    name=ENV_ZENML_LOCAL_STORES_PATH,
-                    value=local_stores_path,
-                )
+            # TODO: Rewrite this security context logic once v2 supports
+            #  security context
+
+            # if sys.platform == "win32":
+            #     # File permissions are not checked on Windows. This if clause
+            #     # prevents mypy from complaining about unused 'type: ignore'
+            #     # statements
+            #     pass
+            # else:
+            #     # Run KFP containers in the context of the local UID/GID
+            #     # to ensure that the local stores can be shared
+            #     # with the local pipeline runs.
+            #     container_op.container.security_context = k8s_client.V1SecurityContext(
+            #         run_as_user=os.getuid(),
+            #         run_as_group=os.getgid(),
+            #     )
+            #     logger.debug(
+            #         "Setting security context UID and GID to local user/group "
+            #         "in Tekton pipelines container."
+            #     )
```

Comment:
This we might want to raise an issue in their GH, or see what the alternative that they propose is. (Or remove the code).